### DISCUSSION TITLE
Do we have a plan for a new release?

### DISCUSSION THREAD

**Discussion by ZhangYet** [bpftrace/bpftrace#3442] [Q&A] [answered]
Could I ask when will we release a new version, since we have many unreleased features?

Btw, I'm eager to use the [for-loop](https://github.com/bpftrace/bpftrace/pull/3014) feature .

**Comment by ajor** [comment, marked as answer]
There's no plan yet for the next release, but for-loops were actually included in the previous major release: [v0.21](https://github.com/bpftrace/bpftrace/releases/tag/v0.21.0)

> Add for-each loops for iterating over map elements
>  https://github.com/bpftrace/bpftrace/pull/3003

**Reply by ZhangYet** [reply]
But why it still in master's Unrelease Change: https://github.com/bpftrace/bpftrace/blob/2a3e2497c518bfd5e0cc7bb6437c17e7a71faca7/CHANGELOG.md?plain=1#L16 ?

**Reply by jordalgo** [reply]
Specifically that change was to enable the use of for-loops in multiple probes in the same script, which we couldn't do before we had libbpf load the bpf programs (meaning only one for-loop was allowed per script in v0.21 release). Perhaps you're blocked on this bug fix which is still unreleased.

**Reply by ZhangYet** [reply]
Thanks
